buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -327,6 +322,7 @@ private static void autoConfigureClusterDependencies(
                 Map<String, Object> projectNotation = new HashMap<>();
                 projectNotation.put("path", unreleasedInfo.gradleProjectPath);
                 projectNotation.put("configuration", esNode.getDistribution().getLiveConfiguration());
+                logger.debug("Cluster {} depends on {}", esNode.getName(), projectNotation);
```

Comment:
Debug logging is so noisy it's basically useless. If I'm _actually_ debugging I'm just going to attach a debugger to the build. If we are going to bury these messages under debug logging we should consider whether they are even valuable.